xml/genheaders.py

```diff
@@ -296,7 +296,7 @@ def makeREstring(list):
         defaultExtensions = 'gles2',                # Default extensions for GLES 2
         addExtensions     = None,
         removeExtensions  = None,
-        prefixText        = prefixStrings + apiEntryPrefixStrings + genDateCommentString,
+        prefixText        = prefixStrings + apiEntryPrefixStrings + genDateCommentString + glextVersionStrings,
```

Comment:
`genDateCommentString` should be removed now since it was added because the version string wasn't present.